server.js

```diff
@@ -21,6 +23,14 @@ const redirectToHome = function (req, res) {
     res.redirect('http://codingblocks.com')
 };
 
+app.use('/admin', (req,res,next)=>{
```

Comment:
collapse the two middlewares into a single app.use call ? 

app.use(path, middleware1, middleware2) works